bothub_nlp_nlu_worker_on_demand/settings.py

```diff
@@ -253,7 +251,20 @@ def get_service(path):
         "yo",
         "za",
         "zu",
-    ],
+    ]
+
+BOTHUB_SPACY_MULTILANG = {
+    "queue": BOTHUB_MULTILANGUAGES,
     "image": "xx-SPACY",
-    "service_name": "xx",
+    "service_name": "xx-SPACY",
+}
+
+BOTHUB_BERT_MULTILANGUAGES = []
+for lang in BOTHUB_MULTILANGUAGES:
+    BOTHUB_BERT_MULTILANGUAGES.append(lang + '-BERT')
```

Comment:
Não tenho certeza se esse for é executado antes da variável ser exportada.
Por precaução faria:
` BOTHUB_BERT_MULTILANGUAGES = [(lang + '-BERT') for lang in BOTHUB_MULTILANGUAGES]`